pkg/clustermanager/discovery.go

```diff
@@ -128,6 +128,7 @@ func processProviderWatchEvents(ctx context.Context, w clusterprovider.Watcher,
 				lcluster.Name = lcName
 				lcluster.Spec.ClusterProviderDescName = providerName
 				lcluster.Spec.Managed = false
+				lcluster.Status.ClusterConfig = event.LCInfo.Config
```

Comment:
Maybe check that config is not an empty string here. We should not update LC to ready if we don't have config.

Reply:
Maybe Watch should send Add only if cluster is ready